### PR TITLE
Update select account

### PR DESCRIPTION
**Pull request by chabroA**
Update request account flow to use the `AccountRequest` component for desktop as well (it is already used
on mobile), to allow user to add an account and to make the flow consistent with the one present on [platform-app-web-browser](https://github.com/LedgerHQ/platform-app-web-browser)

Loading screen
<details>
<summary>Before</summary>

dapp-before

</details>

<details>
<summary>After</summary>

dapp-after

</details>

Select account
<details>
<summary>Before</summary>

dapp-before-2

</details>

<details>
<summary>After</summary>

dabb-after-2

</details>

### CODE REVIEW

**Comment by chabroA** [comment]
⚠️  We need to wait for Vault next release that will include `requestAccount` handling before merging this PR ⚠️

**Review by Justkant — APPROVED** [review]

**Review by RamyEB — APPROVED** [review]

**Pull request merged**
